.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -33,7 +33,7 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Logfile**
-Rubberduck generates extensive logging in TRACE-Level. If no log was created at `%APPDATA%\Rubberduck\Logs`, check your settings. Include this Log for bugreports about the behavior of Rubbberduck
+Rubberduck generates extensive logging in TRACE-Level. If no Log was created at `%APPDATA%\Rubberduck\Logs`, check your settings. Include this Log for bug reports about the behavior of Rubberduck.
```

Comment:
I'm thinking "Log" should be lower-case in both instances instead of upper.

Reply:
Yeah probably right, I was thinking log is for trees and Log was for programming, but really let's just strive for sentence case